subsys/usb/device/usb_descriptor.c

```diff
@@ -237,6 +238,11 @@ static int usb_validate_ep_cfg_data(struct usb_ep_descriptor * const ep_descr,
 			continue;
 		}
 
+		/* Skip already fixed entries */
+		if (*fixed_entries & BIT(i)) {
```

Comment:
Sadly it does not work in composite configuration because endpoint address of alternative interface will not be updated. It also does not work right without this patch because it does not reuse endpoint addresses, and may be fatal with nRF USBD controller if there is alternate interface with an ISO endpoint. Maybe we should document it as known issue and do not waste resources.